catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/ResourceOperations.java

```diff
@@ -411,6 +411,17 @@ ResourceResponse getResource(ResourceRequest resourceRequest, boolean isEnterpri
                 LOGGER.info("Unable to download resource", e);
             }
 
+            if (resourceResponse != null) {
+                // must add the request properties into response properties in case the source forgot to
```

Comment:
Are we sure this is the right way to fix this? If we know what sources are not copying the properties, shouldn't we be fixing those sources instead? Also, what if a source must remove one of the request properties and not send it out? This change is going to prevent that from working.